tests/ua-devtools-evm-hardhat-test/contracts/OmniCounter.sol

```diff
@@ -1,9 +1,287 @@
 // SPDX-License-Identifier: UNLICENSED
+
 pragma solidity ^0.8.22;
 
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { OmniCounterAbstract as OmniCounterImpl } from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/examples/OmniCounterAbstract.sol";
 
-contract OmniCounter is OmniCounterImpl {
-    constructor(address _endpoint, address _delegate) OmniCounterImpl(_endpoint, _delegate) Ownable(_delegate) {}
+import { ILayerZeroEndpointV2, MessagingFee, MessagingReceipt, Origin } from "@layerzerolabs/lz-evm-protocol-v2/contracts/interfaces/ILayerZeroEndpointV2.sol";
```

Comment:
What is this change?

Reply:
When Riley moved oapp over from monorepo, he removed the OmniCounter example.  This contract was just inheriting from the existing OmniCounter, so I embedded it directly.